tests/puppeteer/pages/commonPage.js

```diff
@@ -81,11 +81,11 @@ module.exports = class CommonPage {
     switch (parameter) {
       case 'equal':
         await this.page.$eval(selector, el => el.innerText)
-          .then(text => expect(text.replace(/\s+/g, ' ').trim()).to.equal(textToCheckWith));
+          .then(text => global.expect(text.replace(/\s+/g, ' ').trim()).to.equal(textToCheckWith));
```

Comment:
why no import expect at the top of the file?
```
import {expect} from 'chai';
```

Reply:
if so, we had to import it in each file, so we choose to do it in helper as a global var.

Reply:
Bad programming practice :(

Reply:
Ok, good to know, I will do it in an other dedicated PR (to be very simple to review).
Can you tell why it's a bad practice ?